src/dbt_client/dbtCoreIntegration.ts

```diff
@@ -85,6 +86,32 @@ export class DBTCoreProjectDetection
     private executionInfrastructure: DBTCommandExecutionInfrastructure,
   ) {}
 
+  private getPackageInstallPathFallback(projectDirectory: Uri): string {
+    const dbtProjectFile = path.join(
+      projectDirectory.fsPath,
+      "dbt_project.yml",
+    );
+    if (existsSync(dbtProjectFile)) {
+      const dbtProjectConfig: any = parse(readFileSync(dbtProjectFile, "utf8"));
+      const packagesInstallPath = dbtProjectConfig["packages-install-path"];
+      if (packagesInstallPath) {
+        let packageInstallPathFromYaml;
+        if (path.isAbsolute(packagesInstallPath)) {
+          packageInstallPathFromYaml = packagesInstallPath;
+        } else {
+          packageInstallPathFromYaml = path.join(
+            projectDirectory.fsPath,
+            packagesInstallPath,
+          );
+        }
+        if (existsSync(packageInstallPathFromYaml)) {
+          return packageInstallPathFromYaml;
+        }
+      }
+    }
+    return path.join(projectDirectory.fsPath, "dbt_packages");
```

Comment:
should we also check if `dbt_packages` directory exists?

Reply:
okay, should we create one if it doesn't exist?

Reply:
No need for creation. If no project file exists, there is no project anyway, so not sure how we would even land there.